.travis.yml

```diff
@@ -112,10 +127,12 @@ install:
     mkdir -p ${HOME_BIN_PATH}
   - |
     # install cmake
-    mkdir -p /tmp/cmake-download
-    wget --no-clobber --directory-prefix=/tmp/cmake-download/ https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz
-    tar -C /tmp/ -zxvf /tmp/cmake-download/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz
-    export PATH="/tmp/cmake-${CMAKE_VERSION}-Linux-x86_64/bin:${PATH}"
+    if [[ "${TRAVIS_CPU_ARCH}" != "amd64" ]]; then
```

Comment:
cmake doesn't release arm64 binaries, but [Ubuntu Bionic has cmake 3.10.2](https://packages.ubuntu.com/bionic/cmake) which should be new enough